scrapers/javlibrary.yml

```diff
@@ -1,33 +1,77 @@
-name: javlibrary
+name: JavLibrary
+sceneByFragment:
+  action: scrapeXPath
+  queryURL: https://www.javlibrary.com/en/vl_searchbyid.php?keyword={title}
+  #Note: Use the title of your scene. So if your filename is SSNI-000.mp4, remove the extension, save, then use the scraper
```

Comment:
cleaned up the note part a bit , the rest work fine
```yaml
  # Note: The scraper uses the title from your scene to search for the movie id.
  # Make sure that the title is saved  in the correct form e.g. SSNI-000 ( no extensions or other extra words/characters  should be present) before you use the scrape with function.
  # If javlibrary doesn't work you can replace javlibrary with b47w in the queryURL and sceneScraper scene URL part
```

Reply:
> If javlibrary doesn't work you can replace javlibrary with **any mirrors** in the queryURL ~~and sceneScraper scene URL part~~

I think it's better not to name a mirror since they can change their name at any moment.
For the URL part, i think it's better to always have javlibrary on it because: 
- Javlibrary link will always work, maybe not the scraper but if you got the URL, you already succeed scraping it.
- If you used a mirror, you wanted to use javlibrary first, so your first thought was to have a javlibrary URL.